### PR TITLE
[Ugly WIP] Switch to a CSS Grid layout

### PR DESCRIPTION
**Pull request by nchevobbe**

### CODE REVIEW

File: src/components/Editor/Editor.css [hunk 1]
```diff
@@ -39,11 +39,8 @@
  * BUG https://github.com/devtools-html/debugger.html/issues/63
  */
 .editor-wrapper {
-  position: absolute;
-  height: calc(100% - var(--editor-header-height));
-  width: calc(100% - 1px);
-  top: var(--editor-header-height);
-  left: 0px;
+  grid-row: 2 / 3;
```
Comment (codehag) [inline src/components/Editor/Editor.css]: wow this is great

**Comment by wldcordeiro** [comment]
Whoa this is rad @nchevobbe if you're busy and haven't had a chance to look at this again I'm happy to get the conflicts fixed and help pass CI again. 😄

**Comment by nchevobbe** [comment]
> Whoa this is rad @nchevobbe if you're busy and haven't had a chance to look at this again I'm happy to get the conflicts fixed and help pass CI again. 😄

Yes, I don't have much time to devote to this, but @codehag was interested on working on this. But I guess if you can have a look at tests, it would be a good start. Yuilia was mainly interrested by the CSS changes I guess

**Comment by jasonLaster** [comment]
closing this for now. We can come back after the holidays and revisit.

**Pull request closed without merging**